Package.swift

```diff
@@ -0,0 +1,20 @@
+// swift-tools-version:4.0
+import PackageDescription
+
+let package = Package(
+    name: "ImgixSwift",
+    products: [
+        .library(name: "ImgixSwift", targets: ["ImgixSwift"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/IBM-Swift/CommonCrypto", from: "1.0.0"),
```

Comment:
Prior to merging, I was thinking of forking this repository in our org. Thoughts?

Reply:
Normally I'd be opposed to it, but after checking out the repo I think it's not a bad solution. Seems like a pretty thin wrapper, so it probably won't take too much effort to "maintain".